charts/application/values.yaml

```diff
@@ -366,3 +366,7 @@ initDefaults:
       cpu: 500m
       memory: 100Mi
   restartPolicy: Always
+
+# gives the application access to the k8s API with the defined role ("edit", "view" or "admin")
+k8sAPI:
```

Comment:
would prefer a `rbac` object like this:
```yaml
rbac:
  roleBinding: 
```
maybe even under `serviceAccount`, not quite sure